Rdmp.UI/ScintillaHelper/ScintillaTextEditorFactory.cs

```diff
@@ -120,8 +120,8 @@ public Scintilla Create(ICombineableFactory commandFactory = null, SyntaxLanguag
                     }
                     else
                     {
-                        aff = Path.Combine(currentDirectory, "en_us.aff");
-                        dic = Path.Combine(currentDirectory, "en_us.dic");
+                        aff = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "en_us.aff");
+                        dic = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "en_us.dic");
```

Comment:
There is a more complicated piece of code in `UsefulStuff.GetExecutableDirectory()`

Do you feel that `AppDomain.CurrentDomain.BaseDirectory` is better and will 100% always be better than the above method? if so then we should just change to using just that.  

This PR might also invalidate passing in `currentDirectory` in the first place?